src/models/TrackMapper.php

```diff
@@ -90,7 +111,13 @@ public function getTrackById($track_id, $verbose = false)
         return false;
     }
 
-    public function createEventTrack($data, $event_id)
+    /**
+     * @param array $data
+     * @param int $event_id
+     * @return string
```

Comment:
Please add a blank line after the ```@param```s and move the ```@return``` to the last line of the DocBlock